spikingjelly/activation_based/op_counter/spikesim/config.py

```
@@ -0,0 +1,132 @@
+    **API Language:**
+    :ref:`中文 <SpikeSimEnergyConfig-cn>` | :ref:`English <SpikeSimEnergyConfig-en>`
+
+    ----
+
+    .. _SpikeSimEnergyConfig-cn:
+
+    * **中文**
+
+    SpikeSim 事件驱动能耗估计器的运行时能耗配置。
+
+    默认系数有意与已发布的 ``ela_spikesim.py`` 能耗路径保持一致，
```

Ruff (0.15.12)

[warning] Docstring contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF002)